toolkit/phone-assessment-interviews/email-template-app-first-interview-sched.md

```diff
@@ -7,18 +7,29 @@ phase: phone-assessment-interviews
 title: Template - Email to Applicants Requesting First Interview
 ---
 
-Dear Applicant,
+Hello,
 
-Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from resume review. You're now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will last 30-60 minutes. If you move forward from this interview, you will have one additional subject matter expert qualifications interview. If you're referred for selection, you may be asked to participate in additional interviews with hiring managers.
+Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from the {PRIOR PHASE}. You’re now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will take about {TIME OF INTERVIEW} to complete. 
 
-Here are some key details about the position you've applied for:
+{IF THIS IS ONE OF TWO INTERVIEWS: If you move forward from this interview, you will have one additional subject matter expert qualifications interview.}
+{IF ONLY DOING ONE INTERVIEW: If you’re referred for selection, you may be asked to participate in additional interviews with hiring managers.}
 
-- Your job location would be in `{CITY, STATE}`. There are `{no}` reimbursements for relocation expenses.
+Please also note: the interview will be structured, meaning that questions have been determined in advance. In order to ensure fairness, you will be asked the same questions as all other applicants. Your interviewer will not be able to answer questions about the hiring process or the job you're applying for. If you have questions about the process, please contact me. You will have time to ask questions about the role after you have qualified for the position.
```

Comment:
do we need "please also note:"

Reply:
"you will be able to ask the hiring manager questions about the role..."

Reply:
i really like how you added a line apologizing for the robotic nature of a structured interview. should we add a line to use in case of an async interview too- (for use for async interviews: "While we would prefer to conduct this first interview with you live, given the high volume of applicants the Federal Government receives, the first interview will take place on your own for one of our SMEs to listen to afterward. If you pass this round, you will have a second, live qualifying phone interview. Please understand this helps us to process many more applicants for this position than we would be able to otherwise."

Reply:
I don't think this template should be used for async - that's a whole other type of communication.